_includes/footer.html

```diff
@@ -1,7 +1,7 @@
 <footer>
   <div class="grid-container padding-y-6">
     {% unless page.hide_footer_rule %}
-    <hr class="hr-1-dark footer-hr-alignment"> 
+    <!--<hr class="hr-1-dark footer-hr-alignment"> -->
```

Comment:
**suggestion (non-blocking)**: I think that rather than commenting out unused code, it's better to delete it entirely. We can find it in the git history if we ever want to put it back.

Reply:
Left it in by mistake! 🙃

Reply:
🌳👋🌳 (a-la-homer) This caught my eye in my email so thought I'd chime in – this footer rule shows up on light background pages like the [about page](https://18f.gsa.gov/about/) to separate out from main content. 🌳👋🌳

Reply:
With the light grey background, what do we think? I removed it because it looked funky to me. The example page is white to white, but this is grey to white like on "[Our Work](https://18f.gsa.gov/our-work/)"

Reply:
In that case you can add `hide_footer_rule: true` to the frontmatter of the guides page and it will be hidden.

Reply:
Oh yeah overall I guess I made a global change here testing something out then forgot about it. Anywho, did what Igor suggested.